docs/user-quick-start.md

```diff
@@ -149,22 +149,20 @@ enough resources:
    for the details.
    Tested with version v1.12.0.
 
-1. Install the `virtctl` tool. See
-   [virtctl install](https://kubevirt.io/quickstart_minikube/#virtctl)
-   for the details.
-   Tested with version v1.0.1.
+1. Install the `virtctl` tool.
 
-   ```
-   curl -L -o virtctl https://github.com/kubevirt/kubevirt/releases/download/v1.0.1/virtctl-v1.0.1-linux-amd64
-   ```
-
-   After download completes for `virtctl` issue these commands.
+   Working Example:
 
    ```
-   chmod +x virtctl
-   sudo install virtctl /usr/local/bin
+   curl -L -O https://github.com/kubevirt/kubevirt/releases/download/v1.2.0/virtctl-v1.2.0-linux-amd64
+   sudo install virtctl-v1.2.0-linux-amd64 /usr/local/bin/virtctl
+   sudo rm virtctl-v1.2.0-linux-amd64
    ```
 
+   See
+   [virtctl install](https://kubevirt.io/quickstart_minikube/#virtctl)
+   for more details.
```

Comment:
This is also not in the same style as other nodes around. We use "For more info on ..." in other places. Any reason to use something else here?